apps/web/pages/org/[orgSlug]/[user]/[type]/embed.tsx

```diff
@@ -1,19 +1,7 @@
-import type { GetServerSidePropsContext } from "next";
+import withEmbedSsr from "@lib/withEmbedSsr";
 
 import { getServerSideProps as _getServerSideProps } from "../[type]";
 
 export { default } from "../[type]";
 
-export const getServerSideProps = async (context: GetServerSidePropsContext) => {
```

Comment:
I am not sure why it wasn't using withEmbedSsr, all other embed pages were using that only.